redash/models/__init__.py

```diff
@@ -632,7 +632,7 @@ def past_scheduled_queries(cls):
 
     @classmethod
     def outdated_queries(cls):
-        queries = (
+        queries = list(
```

Comment:
Worth adding a comment, as someone might eagerly refactor the code and remove this. 

I wonder if there is a way to add a test for this somehow?

Reply:
It may be better to append [`all()`](https://docs.sqlalchemy.org/en/13/orm/query.html#sqlalchemy.orm.query.Query.all) to this query statement instead since it's more obvious what it does.

Reply:
I like @jezdez's suggestion. It might even help with @arikfr's concern.

Reply:
👍